simple.cs

```diff
@@ -22,9 +22,11 @@ static void Main(string[] args)
             }
         }
         var ordered = counts.OrderByDescending(pair => pair.Value);
+        StringBuilder stringBuilder = new StringBuilder();
         foreach (var entry in ordered)
         {
-            Console.WriteLine("{0} {1}", entry.Key, entry.Value);
+            stringBuilder.AppendLine($"{entry.Key} {entry.Value}");
```

Comment:
Technically, behind the scenes, `entry.Value` will be boxed. Furthermore, this allocates an entirely new string where atleast 2/3rds of the result already is a string. Proper usage would be to do something like 
```cs
stringBuilder.Append(entry.Key).Append(" ").AppendLine(entry.Value)
```